lib/themes/dosomething/paraneue_dosomething/js/campaign/OrganDonation.js

```diff
@@ -137,6 +139,13 @@ class OrganDonation extends Takeover {
     this.loadScreen(this.$shareScreen);
 
     // @TODO - Handle submission of "done" button that posts to our db info about the submission being done.
+    // This happens whether they hit "done" or "skip" since at this point those are the same
```

Comment:
Ah sorry this is kind of unclear. This is referring specifically to the last page of the modal. It only goes away forever if they hit `Done` or `skip` on the last page.